src/osp/Active/SysPrefabInit.cpp

```diff
@@ -144,9 +144,9 @@ void SysPrefabInit::init_drawing(
 
             }
 
-            MaterialData &rMaterial = rCtxDraw.m_materials[materialId];
-            rMaterial.m_comp.emplace(ent);
-            rMaterial.m_added.push_back(ent);
+            //MaterialData &rMaterial = rCtxDraw.m_materials[materialId];
```

Comment:
Minor: Either remove the commented out code, or add an explanation for why it's being kept.